src/Raven.Server/SqlMigration/Model/MigrationSettings.cs

```diff
@@ -2,10 +2,18 @@
 
 namespace Raven.Server.SqlMigration.Model
 {
+    public enum EmbeddedDocumentSqlKeysStorage
```

Comment:
Please move it to a dedicated `EmbeddedDocumentSqlKeysStorage.cs` file. If it's a public class then let's avoid to put it another .cs file so it's easier to find it via github